docs/ppp/pitch-23112022.md

```diff
@@ -0,0 +1,91 @@
+## The project
+
+**10101** (speaks, **Ten** **Ten** **One**). One App, All Things Bitcoin!
+
+10101 is a non-custodial Lightning wallet infused with financial services such as CFD trading, sports betting and savings.
+
+## The team
+
+We are a group of 5 developers who initially built [ItchySats](https://itchysats.network/) together. Now we are bringing non-custodial financial services to Lightning onto your phone.
+
+- bonomat, aka Philipp
+
+  - [https://github.com/bonomat](https://github.com/bonomat)
+  - [https://makers.bolt.fun/profile/363/bonomat](https://makers.bolt.fun/profile/363/bonomat)
+
+- da-kami, aka Daniel
+
+  - [https://github.com/da-kami](https://github.com/da-kami)
+  - [https://makers.bolt.fun/profile/381/daniel](https://makers.bolt.fun/profile/381/daniel)
+
+- holzeis, aka Richard
+
+  - [https://github.com/holzeis](https://github.com/holzeis)
+  - [https://makers.bolt.fun/profile/408/holzeis](https://makers.bolt.fun/profile/408/holzeis)
+
+- klochowicz, aka Mariusz
+
+  - [https://github.com/klochowicz](https://github.com/klochowicz)
+  - [https://makers.bolt.fun/profile/461/mariusz](https://makers.bolt.fun/profile/461/mariusz)
+
+- luckysori, aka Lucas
+
+  - [https://github.com/luckysori](https://github.com/luckysori)
+  - [https://makers.bolt.fun/profile/534/luckysori](https://makers.bolt.fun/profile/534/luckysori)
+
+## The next steps
+
+![](https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/166aa91f-7136-4f4e-e35e-d4b1a7eefc00/public)
+
+1. Evaluate
+
+   1. With the outcome of the tournament we want to collect feedback from users to learn how to build the right product
+
+2. Build MVP
+
+   1. Once collected, it's all about building the product right. We aim for a ready-to-use MVP in Q1/23
```

Comment:
```suggestion
   1. Once collected, it's all about building the product right. We aim for a ready-to-use `mainnet` MVP in Q1/23
```

I think it would be good to highlight that we want this to be usable with real Bitcoin soon!
I feel you imply that with `ready-to-use`, but I would state it more explicit.